docs/conf.py

```diff
@@ -5,7 +5,11 @@
 # -- stdlib imports ------------------------------------------------------------
 import os
 import warnings
-from datetime import UTC, datetime
+
+# from datetime import UTC, datetime #Feb2025 - comment out
```

Comment:
Is there a need to keep these in the file?

Reply:
Not yet at the moment, will look into file after merging this PR.